### PR TITLE
Create reducer tests

### PR DESCRIPTION
**Pull request by meghanprestemon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "private": true,
   "dependencies": {
     "@databraid/github-widget": "^1.0.16",
-    "@databraid/slack-widget": "^1.0.14",
-    "@databraid/transit-widget": "^1.0.21",
+    "@databraid/slack-widget": "^1.0.17",
```
Comment (HamidAghdaee) [inline package.json]: these shouldn't have ^. 

Also are these the latest versions of these npm modules?

File: package.json [hunk 2]
```diff
@@ -21,7 +21,9 @@
     "eslint-plugin-import": "2.7.0",
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.1.0",
+    "jest": "20.0.4",
```
Comment (HamidAghdaee) [inline package.json]: you shouldn't need jest. Should already be a part of create-react-app

File: package.json (hunk 1 above)
Comment (meghanprestemon) [reply package.json]: Removed ^. As of the submission of this pull request on Friday these were the latest versions.

File: package.json (hunk 2 above)
Comment (meghanprestemon) [reply package.json]: Uninstalled Jest

**Pull request merged**